lib/navigation/guard.dart

```diff
@@ -23,6 +25,21 @@ class BackofficeGuard extends Guard {
     };
   }
 
+  static Future<bool> Function(BuildContext) _hasSelectedApp() {
+    return (BuildContext context) {
+      return Future.delayed(
```

Comment:
instead of delayed a future you can do:

` return Future<bool>.value(context.read...);`